app/src/main/java/com/bnyro/recorder/App.kt

```diff
@@ -6,10 +6,12 @@ import com.bnyro.recorder.util.Preferences
 import com.bnyro.recorder.util.ShortcutHelper
 
 class App : Application() {
+    lateinit var container: AppContainer
```

Comment:
I'd prefer storing the file repository here directly instead of adding an `AppContainer`

Reply:
By the way this is the recommended way of DI
https://developer.android.com/training/dependency-injection/manual#dependencies-container

Reply:
Nevertheless it's unnecessary code.